src/generateRoutes.js

```diff
@@ -71,6 +79,21 @@ function generateRoutes(routes, app, endpointInfo = false) {
         await currentController(usecase, request, req.user, res, next)
       })
     }
+
+    if (route.other) route.other.forEach((other) => {
+      if (other.REST?.verb && other.REST?.path) {
+        const endpoint = `${other.REST.path}`
```

Comment:
There should be a default verb for use cases with `herbarium.crud.other` but without a explicit http verb. My suggestion is to be defaulted to `POST`

Reply:
Fixed